aderyn_core/src/detect/nc/time_units_not_used.rs

```diff
@@ -0,0 +1,77 @@
+use std::{collections::BTreeMap, error::Error};
+
+use crate::{
+    ast::NodeID,
+    capture,
+    context::workspace_context::WorkspaceContext,
+    detect::detector::{IssueDetector, IssueDetectorNamePool, IssueSeverity},
+};
+use eyre::Result;
+
+#[derive(Default)]
+pub struct TimeUnitsNotUsedDetector {
+    // Keys are source file name and line number
+    found_instances: BTreeMap<(String, usize, String), NodeID>,
+}
+
+impl IssueDetector for TimeUnitsNotUsedDetector {
+    fn detect(&mut self, context: &WorkspaceContext) -> Result<bool, Box<dyn Error>> {
+        let disallow = [
+            "second", "seconds", "minute", "minutes", "hour", "hours", "day", "days", "week",
+            "weeks", "month", "months",
+        ];
```

Comment:
When we see variables names that match with this list, we recommend using natively available units like `1 days` , etc in solidity

Reply:
`month` isn't a time unit in Solidity